<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabWidget::pane {
    background-color: #fff;
    border: 1px solid #ccc;
    border-radius: 5px;
    margin-top:5px;
}

QTabBar::tab {
    background-color: #e1e1e1;
    color: #777;
    padding: 8px;
    margin-right:3px;
    border-radius:5px 5px 0 0;
}

QTabBar::tab:selected {
    background-color: #fff;
    color: #000;
	font-weight:bold;
}

QTabBar::tab:last {
   margin-right:0;
}

QLineEdit {
    border:1px solid #ccc;
}

QPushButton {
	border:1px solid #3f89b4;
	border-radius:3px;
    background-color: #3F89B4;
    color: #FFFFFF;
	font-weight:bold;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>380</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>KoPubWorld돋움체 Bold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 104, 156);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>회원 관리 프로그램</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>340</width>
      <height>265</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>로그인</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>아이디 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginid_edit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>55</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>비밀번호 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginpw_edit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>85</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="login_btn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>로그인</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginreset_btn">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>초기화</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>회원가입</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>36</x>
        <y>30</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>아이디 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="joinid_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>25</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>60</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>비밀번호 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="joinpw_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>55</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="joinname_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>86</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>27</x>
        <y>91</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>     이름 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="joinemail_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>118</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>123</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string> 이메일 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="joinage_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>34</x>
        <y>155</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>   나이 :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="join_btn">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>회원가입</string>
      </property>
     </widget>
     <widget class="QPushButton" name="joinreset_btn">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>초기화</string>
      </property>
     </widget>
     <widget class="QPushButton" name="idcheck_btn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>25</y>
        <width>70</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>중복체크</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>회원조회</string>
     </attribute>
     <widget class="QPushButton" name="memberreset_btn">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>초기화</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="memberage_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="memberpw_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>55</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="membersearch_btn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>25</y>
        <width>70</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>조회하기</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>26</x>
        <y>123</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string> 이메일 :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="memberid_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>25</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>아이디 :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="membermodify_btn">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>정보수정</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="membername_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>86</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>91</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>     이름 :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>27</x>
        <y>155</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>   나이 :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="memberemail_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>118</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>23</x>
        <y>60</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>비밀번호 :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>회원탈퇴</string>
     </attribute>
     <widget class="QPushButton" name="exit_btn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>회원탈퇴</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="memberpw_exit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>75</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>아이디 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="memberid_exit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>105</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>비밀번호 :</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>loginid_edit</tabstop>
  <tabstop>loginpw_edit</tabstop>
  <tabstop>login_btn</tabstop>
  <tabstop>loginreset_btn</tabstop>
  <tabstop>joinid_edit</tabstop>
  <tabstop>joinpw_edit</tabstop>
  <tabstop>joinname_edit</tabstop>
  <tabstop>joinemail_edit</tabstop>
  <tabstop>joinage_edit</tabstop>
  <tabstop>join_btn</tabstop>
  <tabstop>joinreset_btn</tabstop>
  <tabstop>idcheck_btn</tabstop>
  <tabstop>memberreset_btn</tabstop>
  <tabstop>memberid_edit</tabstop>
  <tabstop>membersearch_btn</tabstop>
  <tabstop>memberpw_edit</tabstop>
  <tabstop>membername_edit</tabstop>
  <tabstop>memberemail_edit</tabstop>
  <tabstop>memberage_edit</tabstop>
  <tabstop>membermodify_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
